airbyte-integrations/connectors/source-klaviyo/acceptance-test-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+docker run --rm -it \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/tmp \
+    -v $(pwd):/test_input \
+    airbyte/source-acceptance-test:dev \
```

Comment:
this should point to `latest` (or equivalently, have no tag). Otherwise the user has to manually build SAT which is not a great DX for community contributors.

Reply:
yeah, probably copy-pastemistake